Documentation.md

```diff
@@ -1,12 +1,44 @@
-## Documentation
+## Documentation - WIP
 
 * [Terminal Commands](#terminal-commands)
 * [Git Commands](#git-commands)
 * [Glossary](#glossary)
 
 ### Terminal Commands
+- `mkdir DirectoryName` - creates a new directory
+- `cd DirectoryName` - goes to the wished directory
+- `touch Demo.rb` - creates an empty file
+- `echo "Some text" >> Filename.extension` - creates a file with some text in it
+- `cat Filename` - shows what’s inside the file
+- `open Filename` - opens the file from Terminal with the corresponding program/editor to edit the contents as you normally would
+- `ls` - lists everything in the current folder except 'dotfiles'/hidden files (e. g. .gitignore)
+- `ls -a` - lists verything (including the hidden files)
+- `cd ..` - moves up to the parent directory
```

Comment:
For `ls -la` and `cd ..` maybe it is better to create a sub header and describe it. `cd ..` is the same as `cd DirectoryName`. 

Sorry for the super late answer.